client/dist/html/pages/tom_clancy.html

```diff
@@ -1,4 +1,5 @@
 <div id="tom_clancy">
+	<audio autoplay="autoplay" loop="loop"><source src="wup.mp3" type="audio/mpeg"/></audio>
```

Comment:
That wasn't me; the only thing the build script did to dist/ was apply the exact changes that were in client/html/pages.